pkg/controller/config/config_controller.go

```diff
@@ -251,7 +252,8 @@ func (r *ReconcileConfig) Reconcile(req reconcile.Request) (reconcile.Result, er
 func (r *ReconcileConfig) applyPipeline(req reconcile.Request, cfg *op.Config) (reconcile.Result, error) {
 	log := requestLogger(req, "apply-pipeline")
 
-	if err := transformManifest(cfg, &r.pipeline); err != nil {
+	images := toLowerCaseKeys(imagesFromEnv("IMAGE_PIPELINE_"))
```

Comment:
can we move 
`toLowerCaseKeys(imagesFromEnv("IMAGE_PIPELINE_"))`
into the transfor function `transform.DeploymentImages`
and pass `prefix string` as an arg to the `transform.DeploymentImages`